README.md

```diff
@@ -126,10 +126,11 @@ $ oc login --insecure-skip-tls-verify=true -u developer -p developer "${K8S_AUTH
 ```
 
 As we are gentle people, we also provide a shortcut to automate running a new
-cluster and login to it (GNU/Linux only for now):
+cluster and login to it:
 
-```
-$ bin/dev
+```bash
+# Substitute 192.168.1.10 with your local IP that has access to the internet
```

Comment:
only for MAC users, linux users can just run bin/dev...

Reply:
Well, as this feature is still experimental, I prefer a "solution that works everywhere:tm:" for a quick start and not having to deal with particular cases, systems.

Reply:
But this will be detailed in the documentation.